### DISCUSSION TITLE
Disable Suricata IPS

### DISCUSSION THREAD

**Discussion by emaragkos** [Security-Onion-Solutions/securityonion#6369] [Unsupported Versions] [answered]
Hi all, 
I have just installed SecurityOnion and I must say I'm impressed of how well it is made and how easy it is to start using it. Awesome work!

I'm using SecurityOnion to monitor a local lab and expirement with offensive security. The thing is that I'm looking for a way to keep monitoring network/endpoint traffic and get alerts but disable the block/drop of detected "malicious" traffic. I'm interestged in better understand what is happening rather than actually protecting the network. In other words, is there a way to disable the IPS and leave the IDS as it is? Btw I'm using Suricata.

Thanks!

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/faq.html?highlight=ips#can-security-onion-run-in-ips-mode

**Comment by emaragkos** [comment]
Hey Doug, thanks for your answer. Excuse me for asking something that has already been answered in FAQ but it seems like I have probably misunderstood something.
For example, I am trying to exploit a host with MS17-010 and everything works smoothly. When SecurityOnion is active and I try to monitor the traffic, the attack is being detected/identified correctly and probably blocked since the exact same exploitation technique fails. So I guess there is some kind of prevention mechanism active. Correct?

Thanks

**Reply by dougburks** [reply]
If you are sniffing traffic from a tap or span port, then Security Onion is totally passive and has no way to interfere with the traffic.
